app/src/main/java/org/wikipedia/analytics/eventplatform/EventPlatformClient.java

```diff
@@ -171,7 +171,7 @@ static void addEventMetadata(Event event) {
         /*
          * When QUEUE.size() exceeds this value TIMER becomes non-interruptable.
          */
-        private static final int MAX_QUEUE_SIZE = 128;
+        private static final int MAX_OFFLINE_QUEUE_SIZE = 128;
```

Comment:
This no longer has anything to do with online/offline state, does it?  Or am I missing something?

Reply:
Correct! name reverted